auth/prepare.sh

```diff
@@ -1,13 +1,8 @@
-#!/usr/bin/env sh
```

Comment:
You'll want this back as `/bin/bash` does not work on all systems like my nix does not have bash at `/bin`. `/usr/bin/env` is more resilient.... although I don't really see anyone running these outside Docker :man_shrugging:

Reply:
Yeah, they should only run in docker.... :D I know env is good but took a shortcut here 😶